packages/py-api/py_api/routes/utility_routes.py

```diff
@@ -1,28 +1,28 @@
 from typing import Any, Dict
 
-from fastapi import APIRouter, Request
+from fastapi import APIRouter, FastAPI, Request
 from py_api.utilities.memory import get_current_memory_usage_in_mbs
 
+router = APIRouter()
 
-class UtilityRoutes:
-    @staticmethod
-    def bind(router: APIRouter) -> None:
-        @router.get('/health')
-        async def health() -> Dict[str, Any]:
-            return {
-                "status": "healthy",
-                "currentMemoryUsage": f"{get_current_memory_usage_in_mbs()} MB",
-            }
 
-        @router.get('/routes')
-        async def get_routes(request: Request) -> Dict[str, Dict[str, Any]]:
-            routes: Dict[str, Any] = {}
-            for route in request.app.routes:
-                if not routes.get(route.path):
-                    routes[route.path] = [*route.methods]
-                else:
-                    routes[route.path].append(*route.methods)
+@router.get('/health')
+async def health() -> Dict[str, Any]:
+    return {
+        "status": "healthy",
+        "currentMemoryUsage": f"{get_current_memory_usage_in_mbs()} MB",
+    }
 
-            return {
-                "paths": routes,
-            }
+
+@router.get('/routes')
```

Comment:
Nice touch. I see that some of the default routes are listed as well 
![image](https://github.com/AUBGTheHUB/monolith/assets/38375079/a9128438-9274-49f4-b10c-13bfdb42d047)
Do we need them?

Reply:
Do you mean to ask whether we need the endpoint or the docs/openapi routes? If we want to exclude the routes from the list we would need to add explicit filtering. I think it's a good idea to keep them as reminders. I didn't know that they exist before writing the endpoint.

Reply:
We are keeping them.